types/build-schemas.js

```diff
@@ -0,0 +1,233 @@
+"use strict";
```

Comment:
This file shouldn't be included - I've added it to .gitignore in my branch so that it won't be picked up in the future.